### PR TITLE
Add clean code changes

### PR DESCRIPTION
**Pull request by AngelinaTsunik**

### CODE REVIEW

File: JS/test/airport.spec.js [hunk 1]
```diff
@@ -1,103 +1,96 @@
...
-        let flag = false;
-        for (let militaryPlane of bomberMilitaryPlanes) {
-            if (militaryPlane.getMilitaryType() === MilitaryType.BOMBER) {
-                flag = true;
-            }
-            else{
-                assert.fail("Test failed!");
-            }
+      }
+     assert.equal(hasTransportTypeInMilitaryPlane, true);
```
Comment (rimus) [inline JS/test/airport.spec.js]: `assert.isTrue()`?

File: JS/test/airport.spec.js [hunk 2]
```diff
@@ -1,103 +1,96 @@
...
+
+  it('should have at least one bomber in military planes', () => {
+    let airport = new Airport(planes);
+    let bomberMilitaryPlanes  = airport.getBomberMilitaryPlanes();
+    let hasAtLeastOneBomberInMilitaryPlanes = false;
+      for (let militaryPlane of bomberMilitaryPlanes) {
+        if (militaryPlane.getMilitaryType() === MilitaryType.bomber) {
+          hasAtLeastOneBomberInMilitaryPlanes = true;
+        } else {
+          assert.fail("Test failed!");
```
Comment (rimus) [inline JS/test/airport.spec.js]: Poor description.

File: JS/test/airport.spec.js [hunk 3]
```diff
@@ -1,103 +1,96 @@
...
+    it('should check that experimental planes has classification level higher than unclassified', () => {
         let airport = new Airport(planes);
-        let bomberMilitaryPlanes  = airport.getExperimentalPlanes ();
-        let hasUnclassifiedPlanes  = false;
-        for (let experimentalPlane  of bomberMilitaryPlanes) {
-            if (experimentalPlane.classificationLevel === ClassificationLevel.UNCLASSIFIED) {
+        let bomberMilitaryPlanes = airport.getExperimentalPlanes ();
+        let hasUnclassifiedPlanes = false;
+        for (let experimentalPlane of bomberMilitaryPlanes) {
+            if (experimentalPlane.ClassificationLevel === ClassificationLevel.unclassified) {
```
Comment (rimus) [inline JS/test/airport.spec.js]: `experimentalPlane.classificationLevel`?

File: JS/models/classificationLevel.js [hunk 4]
```diff
@@ -1,8 +1,8 @@
...
-    UNCLASSIFIED: 'Unclassified',
-    CONFIDENTIAL: 'Confidential',
-    SECRET: 'Secret',
-    TOP_SECRET: 'Top_secret'
+const CLASSIFICATION_LEVEL = {
+  unclassified: 'Unclassified',
+  confidential: 'Confidential',
+  secret: 'Secret',
+  topSecret: 'Top Secret'
 };
```
Comment (rimus) [inline JS/models/classificationLevel.js]: It is better to use following pattern:
```
const someEnumeration = {
  VALUE_ONE: ...,
  VALUE_TWO: ...,
  VALUE_THREE: ...,
  VALUE_FOUR: ...,
}
```

File: JS/models/experimentalType.js [hunk 5]
```diff
@@ -0,0 +1,8 @@
+const EXPERIMENTAL_TYPE = {
+  liftingBody: 'Lifting Body',
+  hypersonic: 'Hypersonic',
+  highAttitude: 'High Attitude',
+  vtol: 'VTOL'
+};
```
Comment (rimus) [inline JS/models/experimentalType.js]: It is better to use following pattern:
```
const someEnumeration = {
  VALUE_ONE: ...,
  VALUE_TWO: ...,
  VALUE_THREE: ...,
  VALUE_FOUR: ...,
}
```

File: JS/models/militaryType.js [hunk 6]
```diff
@@ -1,7 +1,7 @@
-const MilitaryType = {
-    TYPE_FIGHTER: 'Fighter',
-    TYPE_BOMBER: 'Bomber',
-    TYPE_TRANSPORT: 'Transport'
+const MILITARY_TYPE = {
+  fighter: 'Fighter',
+  bomber: 'Bomber',
+  transport: 'Transport'
 };
```
Comment (rimus) [inline JS/models/militaryType.js]: It is better to use following pattern:
```
const someEnumeration = {
  VALUE_ONE: ...,
  VALUE_TWO: ...,
  VALUE_THREE: ...,
  VALUE_FOUR: ...,
}
```

**Review by rimus — CHANGES_REQUESTED** [review]

File: JS/test/airport.spec.js (hunk 1 above)
Comment (AngelinaTsunik) [reply JS/test/airport.spec.js]: fixed

File: JS/test/airport.spec.js (hunk 2 above)
Comment (AngelinaTsunik) [reply JS/test/airport.spec.js]: fixed

File: JS/test/airport.spec.js (hunk 3 above)
Comment (AngelinaTsunik) [reply JS/test/airport.spec.js]: fixed

File: JS/models/classificationLevel.js (hunk 4 above)
Comment (AngelinaTsunik) [reply JS/models/classificationLevel.js]: fixed

File: JS/models/experimentalType.js (hunk 5 above)
Comment (AngelinaTsunik) [reply JS/models/experimentalType.js]: fixed

File: JS/models/militaryType.js (hunk 6 above)
Comment (AngelinaTsunik) [reply JS/models/militaryType.js]: fixed

**Review by rimus — APPROVED** [review]
Looks good.

**Pull request merged**